core/pom.xml

```diff
@@ -8,18 +8,46 @@
   <version>0.1.0-SNAPSHOT</version>
 
   <name>Census Java Core</name>
+
+  <description>
+    Census provides a framework to define and collect stats against metrics and
```

Comment:
I think we should update this description:

Instrumentation provides a library that includes functionality for tracing, tagging and stats recording.